src/docs/deployment.md

```diff
@@ -189,11 +189,11 @@ Additionally, _if_ you’re writing your [Eleventy Image output](/docs/plugins/i
 
 ### Deploy an Eleventy project to GitHub pages
 
-Includes persisted cache across builds. Using [`peaceiris/actions-gh-pages`](https://github.com/peaceiris/actions-gh-pages).
+Includes persisted cache across builds.
 
 <ol>
 <li>Go to your repository’s Settings on GitHub.</li>
-<li>In the GitHub Pages section change:<ul><li>Source: <code>Deploy from a branch</code></li><li>Branch: <code>gh-pages/(root)</code></li></ul></li>
+<li>In the GitHub Pages section change:<ul><li>Source: <code>GitHub Actions</code></li></ul></li>
```

Comment:
Are you sure you want to deploy for every update of the main branch?

Reply:
It's only meant to be an example so I don't think the branch used here really matters that much. The `build` and `deploy` jobs are the important part. People can change when those jobs are ran to fit their requirements.

Reply:
I like this approach but this changes the what the tutorial was doing ( use a specific build-gh pages script and `peaceiris/actions-gh-pages` ) hence my remark.

Maybe you should rebase this branch since @zachleat merged a related PR to fix an issue.
https://github.com/11ty/11ty-website/pull/1771

I would support the update you propose because to me it look simpler.